src/io/dataset.cpp

```diff
@@ -932,7 +932,7 @@ bool Dataset::GetIntField(const char* field_name, data_size_t* out_len,
 
 void Dataset::SaveBinaryFile(const char* bin_filename) {
   if (bin_filename != nullptr && std::string(bin_filename) == data_filename_) {
-    Log::Warning("Binary file %s already exists", bin_filename);
+    Log::Warning("Bianry file %s already exists", bin_filename);
```

Comment:
It seems that this reverts the typo fixing.

Reply:
Sigh... thanks for spoting this! Fix pushed.